src/store/index.js

```diff
@@ -1,16 +1,50 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 
+import api from "@/api/FactoryRepository.js"
+const moviesApi = api.get("movies")
+
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
+    movies: [],
+    genres: [
+      {name: "Thriller", id: 1},
+      {name: "Comedy", id: 2},
+      {name: "Fantasy", id: 3},
+      {name: "Sci-Fi", id: 4},
+      {name: "Romance", id: 5},
+      {name: "Family", id: 6},
+      {name: "Horror", id: 7},
+      {name: "Documentary", id: 8},
+      {name: "Action", id: 9}
```

Comment:
uuu... you are right - it's missing in the api 🙈

Reply:
I added it with [this PR](https://github.com/monterail/monte-cinema/pull/24) so it should be there by tomorrow

Reply:
Actually, I needed movie genres for dropdown select but im not filtering by genres anymore so problem is gone 🤫 
I guess, still it will be nice that api handle this for future purposes 👊